### ISSUE TITLE
Save video filenames as youtube-ids  

### ISSUE THREAD

**Issue by tohnperfect** [gsssrao/youtube-8m-videos-frames#5]
Hi,

Great work!!

I would like to know if it is possible to save the video filenames using youtube-ids?

**Comment by gsssrao** [comment]
Yes @tohnperfect, you should be able to do that by changing two lines in [downloadvideos.sh](https://github.com/gsssrao/youtube-8m-videos-frames/blob/master/downloadvideos.sh). 

You will have to change the [line 31](https://github.com/gsssrao/youtube-8m-videos-frames/blob/f1261f8d9f098cb2f44db5721cd3f8e14bbda097/downloadvideos.sh#L31) to following:
```
$YTDL -f best "http://www.youtube.com/watch?v=$line" -o ./videos/"%(id)s.%(ext)s"
```

and [line 40](https://github.com/gsssrao/youtube-8m-videos-frames/blob/f1261f8d9f098cb2f44db5721cd3f8e14bbda097/downloadvideos.sh#L40) to following:
```
$YTDL -f best "http://www.youtube.com/watch?v=$line" -o ./videos/"%(id)s.%(ext)s" 2>&1 | tee log.txt
```

Basically, I have just removed `$name%(title)s-` from those lines. The default name format I had set was `CategorynameTitle-YoutubeId.Extension`. Hence by removing `$name%(title)s-`, it will just save the name as `YoutubeId.Extension`.

For more details on formatting youtube-dl output, refer [this link](https://github.com/rg3/youtube-dl#output-template).

Hope this answers your query, let me know if you have any further issues.

**Comment by tohnperfect** [comment]
Thank you very much.
